test/test_main.py

```diff
@@ -33,3 +37,56 @@ def test_mixed(self):
         s = "0 1, 3 5-8, 9 12-10"
         parsed = parse_input_numbers(s)
         self.assertEqual(parsed, [0, 1, 3, 5, 6, 7, 8, 9, 10, 11, 12])
+
+
+class TestMongoDBIntegration(unittest.TestCase):
+    """
+    A few conditional integration test cases with MongoDB.
+
+    It can be configured to not fail the test suite when no MongoDB server
+    is running. In order to provide an instance easily, use Docker or Podman.
+
+    # MongoDB 4
+    docker run -it --rm --publish=27017:27017 mongo:4
+
+    # MongoDB 5
+    docker run -it --rm --publish=27017:27017 mongo:5
```

Comment:
same here

Reply:
Dito. Will be good as soon as GH-8 is merged.